packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
```

Comment:
Ok I'll put it back but I think there's value in trying to get rid of code not needed anymore and maybe not keep un-used code around just in case we might need it in the future.